### PR TITLE
2.11 Release preparation

### PR DESCRIPTION
**Pull request by mayurkathale**
#### 1. Objective

This is the required change to make the release.

**Related information**:

N/A

#### 2. Description of change

Changed version of the plugin in `modules.xml` and `composer.json`
`2.10` => `2.11`
Updated CHANGELOG.md

#### 3. Quality assurance

N/A

**🔧 Environments:**

- **Platform version**: Magento CE 2.3.3.
- **Omise plugin version**: Omise-Magento 2.10.
- **PHP version**: 7.2.16.

#### 4. Impact of the change

N/A

#### 5. Priority of change

Normal

#### 6. Additional Notes

N/A

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+### [v2.11 _(May 08, 2020)_](https://github.com/omise/omise-magento/releases/tag/v2.11)
+
```
Comment (guzzilar) [inline CHANGELOG.md:4]: extra breakline, maybe

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+
+#### 👾 Bug Fixes
+
+- Fixing wrong calculation of installment amount per month showing (PR [#250](https://github.com/omise/omise-magento/pull/250 ))
```
Comment (guzzilar) [inline CHANGELOG.md:8]: ✋  Just minor, the link is working but there is one extra space after `250`

Comment (danfowler) [inline CHANGELOG.md:8]:
```suggestion
- Fixes wrong calculation of installment amount per month (PR [#250](https://github.com/omise/omise-magento/pull/250))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+- Fix webhook to support http POST requests (PR [#254](https://github.com/omise/omise-magento/pull/254))
```
Comment (danfowler) [inline CHANGELOG.md:9]:
```suggestion
- Fixes webhook to support HTTP POST requests (PR [#254](https://github.com/omise/omise-magento/pull/254))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+- Order success page is broken after order checkout (PR [#255](https://github.com/omise/omise-magento/pull/255))
```
Comment (danfowler) [inline CHANGELOG.md:10]:
```suggestion
- Fixes broken order success page after checkout (PR [#255](https://github.com/omise/omise-magento/pull/255))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+### [v2.11 _(May 08, 2020)_](https://github.com/omise/omise-magento/releases/tag/v2.11)
```
Comment (danfowler) [inline CHANGELOG.md:3]: Ideally, we go from down header levels one-by-one, so `#` (H1) to `##` (H2) to `###` (H3).  This skips directly from H1 to H3.

**Pull request merged**